features/step_definitions/email/email_steps.rb

```diff
@@ -0,0 +1,71 @@
+Then(/^I have received an registration complete email/) do
+  if (Quke::Quke.config.custom["urls"]["front_office"]).include? "local"
```

Comment:
You have this line in a number of places, plus in `features/hooks/before.rb` you look at the value in `mail_client` rather than `front_office`.

So just two things

1. Just double check which variable you should be looking at, and if it should be the same make it consistent
1. To help with that, I'd add a method to the `EmailApp` called `local?()` and put this logic in there. Then you can just use `@email_app.local?` when you need to do this check.